Documentation/Extensions/Management.rst

```diff
@@ -105,9 +105,9 @@ Then run `composer require` to the install the local extension:
 
 .. code-block:: shell
 
-   composer require vendor/my-local-extension
+   composer require my-local-vendor/my-local-extension
 
-By executing this command, Composer locates `my-local-extension` and then symlinks
+By executing this command, Composer locates `my-local-vendor/my-local-extension` and then symlinks
```

Comment:
Talking here about an extension key which, so this change is also wrong.

Reply:
It is an example vendor 'my-local-vendor'.
Which vendor should be used here?